demo/python/multiclient-remote-control/client1/.gitignore

```diff
@@ -0,0 +1,2 @@
+# Files generated by running example
+key1.txt
```

Comment:
Add .enc files too?

Reply:
I've added all files in `multiclient-remote-control/data/` to a .gitignore. The `encrypt.py` file encrypts the files to `multiclient-remote-control/data`.